deploy/example/monstorak-storagealert.yaml

```diff
@@ -9,10 +9,10 @@ spec:
     version: "v14.2.1"
     namespace: "openshift-storage"
     serviceMonitor: "rook-ceph-mgr"
-  - provider: "nooba"
+  - provider: "noobaa"
     version: "v1.0"
     namespace: "openshift-storage"
-    serviceMonitor: "nooba-mgr"
+    serviceMonitor: "noobaa-mgr"
```

Comment:
As discussed earlier, this name might be different. But once we start integration testing with NooBaa, we can correct (if required)

Reply:
It is a variable that admin is supposed to change as needed. So, no issues here.